src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/service/EmailSenderService.kt

```diff
@@ -64,25 +71,39 @@ class EmailSenderService(
   }
 
   private fun handleBookedEvent(visit: VisitDto): SendEmailNotificationDto {
-    val templateVars = mutableMapOf(
+    LOG.info("handleBookedEvent email entered")
+
+    val templateVars = getCommonTemplateVars(visit)
+
+    templateVars.putAll(
+      mutableMapOf(
+        "time" to dateUtils.getFormattedTime(visit.startTimestamp.toLocalTime()),
+        "end time" to dateUtils.getFormattedTime(visit.endTimestamp.toLocalTime()),
+        "arrival time" to "45",
+        "closed visit" to (visit.visitRestriction == VisitRestriction.CLOSED).toString(),
+        "visitors" to getVisitors(visit),
+      ),
+    )
+
+    templateVars.putAll(getPrisonContactDetails(visit))
+
+    return SendEmailNotificationDto(templateName = EmailTemplateNames.VISIT_BOOKING, templateVars = templateVars)
+  }
+
+  private fun handleCancelledEvent(visit: VisitDto): SendEmailNotificationDto {
+    LOG.info("handleCancelEmail entered")
```

Comment:
log message

Reply:
Are you saying this should be

    LOG.info("handleCancelledEvent Entered")

?

Reply:
yes, similar to line 74